tests/nfs/generate_report.pm

```diff
@@ -50,6 +51,11 @@ sub display_results {
         my $targs = OpenQA::Test::RunArgs->new();
         $targs->{data} = $test;
         autotest::loadtest("tests/nfs/pynfs_failed.pm", name => $test->{code}, run_args => $targs);
+        $fail = 1;
+    }
+
+    if (!$fail) {
+        record_info('ALL PASSED');
```

Comment:
record_info('ALL PASSED') if (!$fail);

Reply:
Sure, fixed :). I just wonder if wouldn't be more obvious for test result using tests/nfs/pynfs_failed.pm instance writing "ALL PASS".